### ISSUE TITLE
Integrate subdomains with the powerbox.

### ISSUE THREAD

**Issue by zenhack** [sandstorm-io/sandstorm#3595]
I'd like a way to wire up things to a (memorable) subdomain via the powerbox. Example: the current static publishing workflow could be replaced by allowing an app to do a powerbox request, which the shell would recognize and provide the user an option prompting for a subdomain. There are design questions re: how to divide up the namespace (maybe this is something that is admin only by default, and we can write apps to delegate & attenuate, as has been discussed for other admin-only caps). As well as exactly what the interface for this would look like.

**Comment by ocdtrekkie** [comment]
As a note, this may not be a total replacement for the existing functionality: I have hosted a couple static sites on wildly different domains via Sandstorm web publishing. So we may wish to retain the current model as well.

**Comment by zenhack** [comment]
Yeah, just asking for subdomains does not replace hosting on an entirely different domain. But it would still be a nice thing.
